pyiron/lammps/control.py

```diff
@@ -789,6 +789,58 @@ def energy_pot_per_atom(self):
             self['dump___1'] += ' c_energy_pot_per_atom'
             self['dump_modify___1'] = self['dump_modify___1'][:-1] + ' %20.15g"'
 
+    def _set_group_by_id(self, group_name, ids):
+        if len(ids) < 1:
+            raise ValueError('Group ids must have at least length one, but got {}'.format(ids))
+        if np.any([isinstance(id_, bool) or not isinstance(id_, (int, np.int64)) for id_ in ids]):
+            # Note: it turns out bool is a subclass of int. Weird, eh.
+            raise TypeError('Group ids must be integers, but got {}'.format(ids))
+        if np.any(np.array(ids) < 0):
+            raise ValueError('Group ids must be non-negative to be parsed by Lammps, but got {}'.format(ids))
+        self['group___{}'.format(group_name)] = 'id {}'.format(' '.join(np.array(ids).astype(int).astype(str)))
+
+    def _fix_to_three_vector(self, ids, vector, fix_string, conversion):
```

Comment:
shouldn't that be named `_three_vector_to_fix` or `_fix_from_three_vector`?

Reply:
Or `_fix_with_three_vector` or `_three_vector_fix`... 😛 TBH I'm not 100% satisfied with any of these, nor 100% unhappy with any of them. I'll let it stew while the tests run and might rename in the next set of pushes. Since you also provide more than one option I'll assume you're not totally passionate about it either 😉

Reply:
I understand the meaning of all of them, except `_fix_to_three_vector`. That has the opposite meaning of all the others to me, i.e., I'd expect to pass in a fix and get a three vector.